atlassian_jwt_auth/contrib/django/decorators.py

```diff
@@ -1,14 +1,19 @@
+import logging
 from functools import wraps
 
 from django.conf import settings
 from django.http.response import HttpResponse
 from django.utils import six
 from jwt.exceptions import (InvalidIssuerError, InvalidTokenError)
-from requests.exceptions import ConnectionError, HTTPError
+from requests.exceptions import RequestException
 
 import atlassian_jwt_auth
+from atlassian_jwt_auth.exceptions import (PrivateKeyRetrieverException,
+                                           PublicKeyRetrieverException)
 from .utils import parse_jwt, verify_issuers
 
+logger = logging.getLogger(__name__)
```

Comment:
In general I am not a fan of having globals like this. Because this logger is currently only used inside the `requires_asap` method please move it inside that method.